controllers/article_controller.js

```diff
@@ -35,3 +36,19 @@ exports.getArticleComments = (req, res, next) => {
       next(err);
     });
 };
+
+exports.updateArticle = (req, res, next) => {
+  const { article_id } = req.params;
+  const update = req.body;
+
+  Promise.all([
+    updateArticleById(article_id, update),
```

Comment:
we dont need to do a check if article id exists as this can be handled in the model. We can do a Promise.reject in the model for if the rows come back as 0